### PR TITLE
Update illustrations and copy of task list experiment 1

### LINKED ISSUE #32802: [WC 6.5 beta1] Some copy and header images in task list experiment 1 are different from design
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

In the new task list experiment the copy and images for some of the tasks are slightly different. This was missed when initially updating the new task list design.
There is some more context in this thread: p1650896716380659/1650522879.606939-slack-C01CMSA1VHS

It be nice to get this cherry picked into 6.5, but it doesn't break any changes, it could have a slight affect on the experiment results.

See wireframe of Treatment 1 here: G9vi4e286yjdzyIdg8hBb1-fi-735%3A83647
**Note**
Don't include the header for the Store Details task, this is done as part of https://github.com/woocommerce/woocommerce/issues/32760 

### Expected behavior

- Copy and header illustrations should match that of the wireframes (note: some tasks might not require any updates)

Design:  G9vi4e286yjdzyIdg8hBb1-fi-735%3A83647

### Steps to reproduce

1. Install the latest of `trunk`
2. Enable usage tracking through finishing the onboarding wizard
3. Visit Tools > WCA Test Helper > Experiments and enable woocommerce_tasklist_setup_experiment_1_2022_04
4. Visit the home screen and step through the task list finishing each task, notice how the header and copy for each task is unique, but slightly different from that in the designs (linked above).

### WordPress Environment

WooCommerce 6.5 Beta 1

### Isolating the problem

- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [ ] I can reproduce this bug consistently using the steps above.

### PR DESCRIPTION
**Pull request by louwie17**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR add's several new illustrations to use for individual tasks and updates the copy. There was also some minor css changes to match the wireframe more accurately.

Closes #32802 

### How to test the changes in this Pull Request:

1. Checkout branch on fresh site & install the WooCommerce Admin Test Helper
2. Enable usage tracking
3. Complete OBW without any of the free features
4. Visit Tools > WCA Test Helper > Experiments and enable `woocommerce_tasklist_setup_experiment_1_2022_04`
5. Open up the wireframe in a separate window G9vi4e286yjdzyIdg8hBb1-fi-735%3A83647 under **Treatment 1**
6. Now with the wireframe side by side, complete each task in the task list and match the copy and illustration of each task (you can skip the uncompleted store details version, there is a separate issue for that).
7. When you completed the task list fully, click the Store details task and step through the onboarding installing **WooCommerce Payments** as part of the free features this time.
8. Go back to the Homescreen and check if the **WooCommerce Payments** task matches that of the wireframe (the header that is, as we have another issue for the ToS within the actual task as displayed in the wireframe - https://github.com/woocommerce/woocommerce/issues/32762 ).
9. Visit Tools > WCA Test Helper > Experiments and enable `woocommerce_tasklist_setup_experiment_2_2022_04` and disabling the other experiment.
10. Go back to the home screen and check if the headers of each panel still look correct.

### Other information:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully run tests with your changes locally?
* [ ] Have you created a changelog file by running `pnpm nx affected --target=changelog`?

### FOR PR REVIEWER ONLY:

* [ ] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

File: plugins/woocommerce-admin/client/two-column-tasks/headers/appearance.js [hunk 1]
```diff
@@ -8,17 +8,24 @@ import { __ } from '@wordpress/i18n';
  * Internal dependencies
  */
 import TimerImage from './timer.svg';
-import PersonalizeMyStore from './illustrations/personalize-my-store.js';
```
Comment (octaedro) [inline plugins/woocommerce-admin/client/two-column-tasks/headers/appearance.js:11]: Should we delete the [illustrations folder](https://github.com/woocommerce/woocommerce/tree/370601c3d49f734c3f12eab08bf12c20e53e2b34/plugins/woocommerce-admin/client/two-column-tasks/headers/illustrations)?

**Comment by octaedro** [comment]
I had an issue with step 7 (experiment `woocommerce_tasklist_setup_experiment_1_2022_04`), after installing WC-Payments through the OBW, nothing happens after pressing `Get paid with WooCommerce Payments`.

screenshot-new local-2022 04 28-12_47_29

**Review by octaedro — COMMENTED** [review]
Good job @louwie17! This is working mainly well here and the code looks good to me. I just left a couple of comments.

File: plugins/woocommerce-admin/client/two-column-tasks/headers/appearance.js (hunk 1 above)
Comment (louwie17) [reply plugins/woocommerce-admin/client/two-column-tasks/headers/appearance.js:11]: Yeah, I originally left it, incase we were going to revert, but that is what Git is for... so deleting it makes the most sense.

**Comment by louwie17** [comment]
> I had an issue with step 7 (experiment `woocommerce_tasklist_setup_experiment_1_2022_04`), after installing WC-Payments through the OBW, nothing happens after pressing `Get paid with WooCommerce Payments`.

@octaedro that is actually expected behaviour (I should of clarified in the test), as it would enforce the user to click `Verify details` in the header instead. Now we are also changing this behaviour as part of this PR that Joel has been working on https://github.com/woocommerce/woocommerce/pull/32824

I will delete the illustrations folder though.

File: plugins/woocommerce-admin/client/two-column-tasks/headers/appearance.js (hunk 1 above)
Comment (louwie17) [reply plugins/woocommerce-admin/client/two-column-tasks/headers/appearance.js:11]: Removed here: https://github.com/woocommerce/woocommerce/pull/32805/commits/711e0ce44935ed54586f91119779846f4a563508

**Comment by louwie17** [comment]
@octaedro I updated the PR if you have time to re-check it. It would probably be worth stepping through the testing instructions again, just incase an illustration was still being used somewhere (I tried locally and it seemed fine).

**Comment by louwie17** [comment]
Actually @octaedro I have one more slight change to make to this PR, will update shortly.

**Comment by louwie17** [comment]
Alright @octaedro, it's formally good for a re-review. I made some minor CSS changes to the individual task list heights to match the design, not sure if this was missed originally or got removed along the way.

**Review by octaedro — APPROVED** [review]
Thank you @louwie17 for the changes. This tested well and LGTM 🚀

**Pull request merged**